src/webgpu/web_platform/canvas/getCurrentTexture.spec.ts

```diff
@@ -14,6 +14,15 @@ const kFormat = 'bgra8unorm';
 class GPUContextTest extends GPUTest {
   initCanvasContext(canvasType: CanvasType = 'onscreen'): GPUCanvasContext {
     const canvas = createCanvas(this, canvasType, 2, 2);
+    if (canvasType === 'onscreen') {
+      // To make sure onscreen canvas are visible
+      const onscreencanvas = canvas as HTMLCanvasElement;
+      onscreencanvas.style.position = 'fixed';
+      onscreencanvas.style.top = '0';
+      onscreencanvas.style.left = '0';
+      // Set it to transparent so that if multiple canvas are created, they are still visible.
+      onscreencanvas.style.opacity = '50%';
```

Comment:
Canvas also needs to be added to the document.

Reply:
Is there a preferred place to appendChild?
I do `document.body.appendChild(onscreencanvas);` now. And in developer tools element view seems canvas are added to the end of the body and that seems okay?

Reply:
I think that's fine.

Also I forgot - we need to remove the canvas after the test is done.

Reply:
I think this could be done with `this.trackForCleanup({ close() { document.body.removeChild(onscreencanvas); } });`